lib/KeyboardAwareMixin.js

```diff
@@ -52,21 +52,24 @@ const KeyboardAwareMixin = {
       if (!currentlyFocusedField) {
         return
       }
-
-      UIManager.viewIsDescendantOf(
-        currentlyFocusedField,
-        this.getScrollResponder().getInnerViewNode(),
-        (isAncestor) => {
-          if (isAncestor) {
-            // Check if the TextInput will be hidden by the keyboard
-            UIManager.measureInWindow(currentlyFocusedField, (x, y, width, height) => {
-              if (y + height > frames.endCoordinates.screenY) {
-                this.scrollToFocusedInputWithNodeHandle(currentlyFocusedField)
-              }
-            })
+      if (Platform.OS == 'android') {
+        this.scrollToFocusedInputWithNodeHandle(currentlyFocusedField)
```

Comment:
This is going to fail in Android if there are in the same scene multiple `KeyboardAwareScrollView`s. All will receive the event but the ones that doesn't include the input as a descendant will crash.